src/components/learner-credit-management/assignment-modal/AssignmentAllocationHelpCollapsibles.jsx

```diff
@@ -78,23 +78,22 @@ const AssignmentAllocationHelpCollapsibles = ({ enterpriseId, course }) => (
           <li>
             <FormattedMessage
               id="lcm.budget.detailsPage.catalog.tab.course.card.total.assignment.cost"
-              defaultMessage="The total assignment cost will be earmarked as {doubleQoute}assigned{doubleQoute} funds in your
-               Learner Credit budget so you can{apostrophe}t overspend."
+              defaultMessage="The total assignment cost will be earmarked as {doubleQuote}assigned{doubleQuote} funds in your
+               Learner Credit budget so you can't overspend."
               description="A step which explains that the total assignment cost will be earmarked as 'assigned' funds in your Learner Credit budget"
               values={{
-                apostrophe: "'",
-                doubleQoute: '"',
+                doubleQuote: '"',
```

Comment:
nit: same comment as above regarding `doubleQuote` (i.e., message arg likely can be removed in favor of escaping the `"` character in the `defaultMessage` string).